kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala

```diff
@@ -82,40 +144,31 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
       case t: Throwable =>
         throw new NotAllowedException(t.getMessage)
     }
+    userName
+  }
 
+  def getEngineSpace(
+      userName: String,
+      engineType: String,
+      shareLevel: String,
+      subdomain: String,
+      subdomainDefault: String,
+      conf: KyuubiConf): String = {
```

Comment:
move this method to `class AdminResource` and `conf` is not needed to be transferred.